### ISSUE TITLE
[Moderation] Setting package to be excluded from testing doesn't save permanently (or does it?)

### ISSUE THREAD

**Issue by flcdrg** [chocolatey/home#53]
1. Click the 'Exempt the package from verification testing' checkbox and add a reason
2. Leave package status as 'Submitted'
3. Clear 'If Submitted status, require maintainer to make changes?'
4. Hit **Save**
5. Page reloads and message pop-up appears saying changes have been saved (and Verification Testing Exemption is shown) 
6. Go to browser location bar, click and press enter (to reload page without re-posting)
7. Page shows no testing exemption

**Comment by flcdrg** [comment]
Hmm.. tried it with another package and didn't clear the 'If Submitted status, require maintainer to make changes` flag, and it seems to have stuck that time.

**Comment by flcdrg** [comment]
Yep.. just confirmed - if I check 'If Submitted status, require maintainer to make changes', then it doesn't save.

That would be the bug.

**Comment by flcdrg** [comment]
And because that check is not cleared, the package is now in a 'Waiting for maintainer' state, which is not ideal

**Comment by gep13** [comment]
Not sure when I will get a chance to look at this one, but I will try as soon as I can.

**Comment by flcdrg** [comment]
Looks like a workaround is to make sure you add a moderation comment. If I do that, then the changes seem to stick
